### ISSUE TITLE
Linear-gradient background does not work

### ISSUE THREAD

**Issue by Cisneiros** [astefanutti/decktape#61]
I've been struggling to export my Remark presentation. My slides have backgrounds using CSS3 Linear Gradients, and these appear not to work with decktape. This is the CSS I use for the slides:

``` css
.remark-slide-content.blue {
    background: #7474BF;
    background: -webkit-linear-gradient(-30deg, #7474BF, #348AC7);
    background: linear-gradient(-30deg, #7474BF , #348AC7);
    color: #fdfdfd;
}
```

All my slides are exported with white background. If I remove the linear gradient (both with and without `-webkit` prefix), it works (I get the solid BG color as specified). If I leave the gradient (I've tried leaving only the prefixed one, the not prefixed one, and both), I get a white background.

On a side note (and possibly another bug, but I don't want to get into it right now, it just helps eliminate some things on this issue), PhantomJS seems to render linear gradients. My slides used to have `animate.css` fade-in transitions. When I have these on, my slides are exported as raster images (bitmaps) inside the PDF, instead of text. In this case, I get the linear background just fine (but my slides are now images instead of texts). When I remove the animations, I get regular text inside the PDF, but no background.

I'm pretty lost, I must say!

**Comment by astefanutti** [comment]
I confirm I can reproduce the issue with standard PhantomJS example, by executing `phantomjs examples/rasterize.js test.html test.pdf`, using `test.html`:
```
<html>
<head>
<title>test CSS linear-gradient</title>
</head>
<style>
    .gradient {
        background: #7474BF;
        background: -webkit-linear-gradient(-30deg, #7474BF, #348AC7);
        background: linear-gradient(-30deg, #7474BF , #348AC7);
        color: #fdfdfd;
    }
    .full {
        height:100%;
    }
</style>
<body class="full">
    <div class="gradient full">
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </div>
</body>
</html>
```

Note that this is specific to the PDF output, as executing `phantomjs examples/rasterize.js test.html test.png` works fine.

CSS3 support is known to be lagging in PhantomJS due to the old version of WebKit used by Qt WebKit. Hopefully this will be solved with ariya/phantomjs#14458 (PhantomJS 2.5, which will use a recent version of WebKit). Though it may still require some work at the level of Qt WebKit.

For the second issue you mention, I noticed that text with alpha layer is rendered rasterised instead of vector text. So it seems in some use cases, the rendering falls back to bitmap rasterising instead of proper PDF primitives.

**Comment by astefanutti** [comment]
Adding ariya/phantomjs#14492 for reference.

**Comment by astefanutti** [comment]
Should be fixed in version `2.0.1` that uses headless Chrome. Let me know if you still face this issue.
